src/authorizations/components/redesigned/IndividualAccordionBody.tsx

```diff
@@ -26,6 +26,15 @@ interface Props {
 
 export const IndividualAccordionBody: FC<Props> = props => {
   const {resourceName, permissions, onToggle, title, disabled} = props
+  let sortedPermissions
+
+  if (title === 'Individual Bucket Names') {
+    // re-order buckets: user buckets first followed by system buckets
+    const systemBuckets = permissions.splice(0, 2)
```

Comment:
do system buckets always come back at the front of the array here? what's the original source they come from? is it possible to sort them there in case they need to be sorted elsewhere in the tokens section?

if this needs to change, I'll re-review it

Reply:
![Screen Shot 2021-11-22 at 5 13 01 PM](https://user-images.githubusercontent.com/66275100/142948796-2174267e-4c97-44f7-a2e7-e016c2060ac7.png)
the bucket objects are pushed inside an array and then passed to this Filter component you see in the screenshot. then the functions inside Filter sort the buckets by name and return them to IndividualAccordionBody. My thought process here was - We know system buckets are the only buckets that start with '_'. User buckets can't be named with an underscore. So when being sorted, the two strings that start with '_' will always take precedence....

Reply:
I can also loop through permissions array inside `IndividualAccordionBody` and filter out system buckets. and then re-order them in a new array. 
that way we don't have to rely on "system buckets will always take the first and second index" theory.